recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -175,11 +177,12 @@ def upload_package(feedstock_root, recipe_root, config_file, validate, feedstock
         if validate and owner == "conda-forge":
             retry_upload_or_check(
                 feedstock_name, recipe_root, STAGING, channel,
-                [config_file], validate=True, git_sha=git_sha)
+                [config_file], validate=True, git_sha=git_sha,
+                private_upload=private_upload)
```

Comment:
You won’t ever need this flag passed here.

Reply:
True! :-)